src/Worker/BuildWorker.php

```diff
@@ -18,6 +18,7 @@
 class BuildWorker
 {
     const JOB_TYPE = 'php-censor.build';
+    const JOB_ID_STOP = '.php-censor.worker.stop';
```

Comment:
`'php-censor.stop-flag'` would be a better name for me.

Reply:
👍